main.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+class rs_m

```

Comment:
This class name looks a bit off, I believe the c++ class naming standard is "CamelCase". Also, it's not apparently clear what the class is supposed to do, maybe you should name it something like "ResortManagement"?